src/python/kiss_icp/pipeline.py

```diff
@@ -154,7 +154,7 @@ def _save_poses(self, filename: str, poses: List[np.ndarray]):
 
     def _write_result_poses(self):
         self._save_poses(
-            filename=f"{self.results_dir}/{self._dataset.sequence_id}_poses",
+            filename=f"{self.results_dir}/{self.dataset_sequence}_poses",
```

Comment:
here it was a small bug when not a sequence dataloader reaches this point and fails nbecause there is no sequence_in the dataset.

Reply:
Good catch!